packager/packager.cc

```diff
@@ -95,7 +95,9 @@ class PackageDB final {
             packageInfoByMangledName[pkg->name.mangledName] = pkg;
         }
 
-        packageInfoByPathPrefix[pkg->packagePathPrefix] = pkg;
+        for (std::string packagePathPrefix : pkg->packagePathPrefixes) {
```

Comment:
Please make this `for (const std::string &packagePathPrefix : ...` so at least we're only copying the string once to make the hash key rather than twice.